ansible/roles/ci/doc/how_does_it_work.md

```diff
@@ -0,0 +1,12 @@
+# How does it work
+
+The build tools was created to run on the [Docker](https://www.docker.com/) based CI servers.
+The Docker image for all build is one hosted on [Docker Hub](https://hub.docker.com/r/shadowrobot/ubuntu-ros-indigo-build-tools)
+
+The build script is loading Ubuntu 14.04 Docker image with pre-installed ROS Indigo.
+This image is dependent on Ubuntu and ROS repositories in Docker Hub which means that image is rebuild as soon as any of parent images changes.
```

Comment:
"This image is dependent on Ubuntu and ROS repositories in the Docker Hub, which means that the image is rebuild as soon as any of parent images change"